package.json

```diff
@@ -1298,7 +1298,7 @@
             "description": "%cmake-tools.configuration.cmake.searchDirs.items.description%"
           },
           "default": [
-            "C:\\MinGW"
+            "${env.SystemDrive}/MinGW"
```

Comment:
VS Code variable syntax asks for `env:` instead of `env.`.  We support both because VS did it with `.` instead of `:` and people get confused.  But for VS Code we should use a colon `:` if we set this as a default.